docs/3.api/3.utils/navigate-to.md

```diff
@@ -8,16 +8,16 @@ links:
     size: xs
 ---
 
-::note
-`navigateTo` is available on both client and server side (but not within Nitro routes).
-::
-
 ## Usage
 
 `navigateTo` is available on both server side and client side. It can be used within the [Nuxt context](/docs/guide/going-further/nuxt-app#the-nuxt-context), or directly, to perform page navigation.
 
-::tip
-To send a redirect from a server endpoint, use [`sendRedirect`](https://h3.unjs.io/utils/response#sendredirectevent-location-code) instead.
+::warning
+Make sure to always use `await` or `return` on result of `navigateTo` when calling it.
```

Comment:
IMO this literally pinpoint a DX issue. Users shouldn't think about "should i use `await` or `return` here ?" .

@danielroe should we do something about this in V4 ?

Reply:
we probably could (in v3) if we _require_ `defineNuxtRouteMiddleware` wrapper - because we could 'catch' the value and return it.

(basically vue-router middleware redirects via a 'returned value' but elsewhere `navigateTo` navigates via a push/server redirect.)